test/e2e/suite_test.go

```diff
@@ -864,8 +870,8 @@ func addDcToCluster(t *testing.T, ctx context.Context, namespace string, f *fram
 
 	t.Log("check that nodes in dc1 see nodes in dc2")
 	pod := "test-dc1-default-sts-0"
-	count := 4
-	checkNodeToolStatus(t, f, f.K8sContext(0), namespace, pod, count, 0, "-u", username, "-pw", password)
+	count := dcSize * 2
+	checkNodeToolStatus(t, f, "kind-k8ssandra-0", namespace, pod, count, 0, "-u", username, "-pw", password)
```

Comment:
Should not hardcode kind-k8ssandra-0 here.

Reply:
yes, good point. That part got modified by another PR after I pushed this one I think.